tracer/src/Datadog.Trace/Debugger/Configurations/ConfigurationUpdater.cs

```diff
@@ -82,7 +82,8 @@ private ProbeConfiguration ApplyConfigurationFilters(ProbeConfiguration configur
                 ServiceConfiguration = configuration.ServiceConfiguration,
                 LogProbes = Filter(configuration.LogProbes, MaxAllowedLogProbes),
                 MetricProbes = Filter(configuration.MetricProbes, MaxAllowedMetricProbes),
-                SpanProbes = Filter(configuration.SpanProbes, MaxAllowedSpanProbes)
+                SpanProbes = Filter(configuration.SpanProbes, MaxAllowedSpanProbes),
+                SpanDecorationProbes = Filter(configuration.SpanDecorationProbes, MaxAllowedSpanProbes)
```

Comment:
Copy-pasta: you should create a dedicated MaxAllowed* for the Span Decoration instead of `MaxAllowedSpanProbes`

Reply:
Fixed in c84d482233d0cffc75af6ff48f08583e125fdcc0